TestAssets/ProjectsWithTests/MultipleFrameworkProject/GivenThatIWantSomeFakeTests.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace FakeTests
+{
+    public class GivenThatIWantSomeFakeTests
+    {
+#if NET451        
+        [Fact]
+        public void NET451_succeeds()
+        {
+            Assert.True(true);
+        }
+        
+        [Fact(Skip="Skipped for NET451")]
+        public void SkippedTest()
```

Comment:
Maybe these two Skipped tests could have different names - so you can verify they are skipped on the right TFM.